github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-presubmits.yaml

```diff
@@ -1,11 +1,12 @@
 presubmits:
   kubevirt/containerized-data-importer:
   - name: pull-cdi-unit-test
+    skip_branches:
+      - release-v1.38
+      - release-v1.43
+      - release-v1.49
     cluster: prow-workloads
-    annotations:
-      fork-per-release: "true"
```

Comment:
For the presubmits for the main branch - its probably worth leaving `fork-per-release` in case CDI decides to use the config-forker for new releases. 
https://github.com/kubernetes/test-infra/tree/master/releng/config-forker#readme

Reply:
OK, that change is omitted.

Reply:
Done